packages/mobile/src/identity/privateHashing.ts

```diff
@@ -1,5 +1,5 @@
 import { ContractKit } from '@celo/contractkit'
-import { getPhoneHash, isE164Number, PhoneNumberUtils } from '@celo/utils/src/phoneNumbers'
+import { getPhoneHash, isE164Number, PhoneNumberUtils } from '@celo/utils/lib/phoneNumbers'
```

Comment:
The mobile app intentionally uses `src`. Tbh I can't remember why but I remember we did this on purpose. @cmcewen do you remember? Was it for better typescript support?

Reply:
I think just because it's the actually TS source as opposed to the built source, so no need to rebuild when making changes